polars/src/lazy/frame.rs

```diff
@@ -747,6 +765,18 @@ mod test {
         assert_eq!(new.shape(), (1, 2));
     }
 
+    #[test]
+    fn test_simplify_expr() {
+        // Test if expression containing literals is simplified
+        let df = get_df();
+        let plan = df
+            .lazy()
+            .select(&[lit(1.0f32) + lit(1.0f32) + col("sepal.width")])
+            .describe_optimized_plan()
+            .unwrap();
+        assert_eq!(plan, "SELECT [LIT Float32(2.0) Plus COLUMN sepal.width] \nFROM\nSELECT [COLUMN sepal.width] \nFROM\nTABLE: [\"sepal.length\", \"sepal.width\", \"petal.length\", \"petal.width\"]");
```

Comment:
Could we test on the Expr structs itself? I plan to modify the string representation of the plan and I don't want this test failing because of that.

Reply:
Yes, will do!